sofiadinizms/NFT-onepage
# Feat/basic sections

## What I did

1. Fixed and added front-end configurations (mainly TS related)
2. Created the component for the basic section layout
3. Created the component that combines and maps all the basic sections

## How to test it
1. Install de project dependencies (just to make sure, I recommend using both `yarn` and `npm install`) on both the `client` and `server` folders
2. Comment the map function on the `src/components/GenericSections/GenericSections.tsx` file
3. Run the front end with `npm start` or `yarn start` and you should be able to view the component

## Technical debts
1. The map function is still not working
2. The section background is yet to be added as a Prop of the Section component so that it can be althered using the Theme colors
